packages/abi-typegen/test/fixtures/forc-projects/index.ts

```diff
@@ -30,12 +32,20 @@ export enum AbiTypegenProjectsEnum {
   VECTOR_SIMPLE = 'vector-simple',
 }
 
-export const getTypegenForcProject = (
+export const getTypegenForcProject = <T extends boolean = false>(
   project: AbiTypegenProjectsEnum,
-  build: 'release' | 'debug' = 'release'
-) =>
-  getForcProject<JsonAbi>({
+  transpile: T = false as T
+) => {
+  const result = getForcProject<T extends true ? JsonAbiOld : JsonAbi>({
     projectDir: join(__dirname, project),
     projectName: project,
-    build,
+    build: 'release',
```

Comment:
Are we still building debug+release somewhere, and is this parameter still necessary?

Reply:
We are not. The rationale behind the flag is explained in https://github.com/FuelLabs/fuels-ts/pull/1744#discussion_r1485614109. I'd leave it, it proved useful for `abi-typegen` supporting the `--json-abi-with-callpaths` flag; it might be useful in the future as well.

Reply:
Feb 11th ref it is then. 🎯 😆